client/src/Components/Login.jsx

```diff
@@ -23,42 +31,74 @@ export default function Login({ user, setUser }) {
     setLoginData({ ...loginData, [e.target.id]: e.target.value });
   };
 
-  const handleCheck = () => {
-    const token = Cookies.get('authToken');
-    const userData = Cookies.get('username');
-    console.log('user: ', user);
-    console.log('token: ', token);
-    console.log('userCookie: ', userData);
+  const goBack = () => {
+    navigate(-1);
   };
 
   return (
-    <div>
-      <form>
-        <label htmlFor="email">
```

Comment:
why the removal of the labels? labels should be added for accessiblity reasons and clarity. Likely should be a new component of LabelledInput but can be a refactor pr to keep things small imo.

Reply:
thats ok i think we will need someone with more insite into accessibility design on this than lil ol me. Thanks for clarifying no changes needed.